recipes/libcap/all/conanfile.py

```diff
@@ -26,82 +30,81 @@ class LibcapConan(ConanFile):
         "psx_syscals": False,
     }
     exports_sources = "patches/**"
+    generators = "VirtualBuildEnv", "VirtualRunEnv"
```

Comment:
why?

Reply:
Hm, I was sure that the migrating guide recommends adding them, because they are enabled by default in v2. But I checked it, and it looks like I just mixed it up.
I will delete it.